client/my-sites/stats/stats-module-utm-builder/stats-module-utm-builder-form.tsx

```diff
@@ -54,6 +57,12 @@ const UtmBuilder: React.FC = () => {
 		utm_medium: '',
 		utm_campaign: '',
 	} );
+	// Focus the initial input field when rendered.
+	const initialInputReference = useRef< HTMLInputElement >( null );
+
+	useEffect( () => {
+		initialInputReference.current!.focus();
```

Comment:
I'm wondering the use of `!.`, can you please explain more why it's used in here? Thanks

Reply:
It's used to quiet the TypeScript warning that `current` might be `null`. Because we know it will be set before the `useEffect` block is run (the JSX for the reference isn't part of any conditional), we can silence the warning like this. Another option would be to manually test before access:

```
if ( initialInputReference.current !== null ) {
  initialInputReference.current.focus();
}
```